recipes/grpclib/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "grpclib" %}
+{% set version = "0.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grpclib-{{ version }}.tar.gz
+  sha256: 8c0021cd038634c268249e4cd168d9f3570e66ceceec1c9416094b788ebc8372
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - protoc-gen-python_grpc=grpclib.plugin.main:main
+    - protoc-gen-grpclib_python=grpclib.plugin.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dataclasses  # [py<37]
+    - h2 <5,>=3.1.0
+    - multidict
+    - python
+
+test:
+  imports:
+    - grpclib
+    - grpclib.channelz
+  commands:
+    - pip check
+  requires:
+    - pip
+    - protobuf
```

Comment:
Why do you require `protobuf` in the tests but not in `run`?

Reply:
Good catch, thanks. This was a leftover from a more expanded set of manual tests against the protobuf generators, as I hadn't used them via `conda` before, and is no longer needed.